src/modules/universal-discovery/components/content-meta-preview/content.meta.preview.component.js

```diff
@@ -56,12 +56,9 @@ export default class ContentMetaPreviewComponent extends Component {
             return;
         }
 
-        const version = response.View.Result.searchHits.searchHit[0].value.Content.CurrentVersion.Version
+        const version = response.View.Result.searchHits.searchHit[0].value.Content.CurrentVersion.Version;
         const imageField = version.Fields.field.find(field => field.fieldTypeIdentifier === 'ezimage');
-        const translations = version.VersionInfo.VersionTranslationInfo.Language.map(langauge => {
-            // @TODO: pass it in props when COTF is cleaned
-            return window.eZ.adminUiConfig.languages.map[langauge.languageCode].name;
-        });
+        const translations = version.VersionInfo.VersionTranslationInfo.Language.map(langauge => this.props.languages.map[langauge.languageCode].name);
```

Comment:
To shorten the line length, please create a variable: `const versionLanguages = version.VersionInfo.VersionTranslationInfo.Language;`